internal/prometheus/prometheus.go

```diff
@@ -0,0 +1,76 @@
+package prometheus
+
+import (
+	"net/http"
+
+	"github.com/AdguardTeam/AdGuardHome/internal/stats"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+// Config - Prometheus configuration
+type Config struct {
+	Enabled   bool   `yaml:"enabled"`
+	Namespace string `yaml:"namespace"`
+
+	AnonymizeClientIP bool `yaml:"-"` // anonymize clients' IP addresses - set from DNS config
```

Comment:
I don't see how it's used further

Reply:
Oops, a left over from when developing which should've been cleaned up :)